[16.0][ADD] stock_picking_report_valued_delivery: Fix delivery slip report if product has a HS Code (#2)
tuantrantg/stock-logistics-reporting: stock_picking_report_valued_delivery/readme/DESCRIPTION.rst

> Valued Picking Report works fine, but when Delivery is installed, there is a conflict between the inheritance introduced by Delivery on stock.report_delivery_document and the one introduced by Valued Picking Report.
> As a consequence, when a product has an HS code defined, the report columns are mixed.
> This module's fixed this issue.

Reviewer: ```suggestion
This module extends `stock_picking_report_valued` to make sure that the columns of Valued Picking Report are not mixed when the Delivery module is installed.
```
